app/utils.py

```diff
@@ -154,3 +154,14 @@ def purge_nulls_from_data(data):
 def get_request_page_questions():
     json_payload = get_json_from_request()
     return json_payload.get('page_questions', [])
+
+
+def validate_user_can_access_direct_award_project_or_403(user_id, project_id):
```

Comment:
We do store it, so we definitely need it. But I'm wondering whether we should be validating auth/access or not.

I really definitely still do think we need to have a way to start putting that logic into the API, but it can be a separate PR once we start on the multi-user thing. So we should try and have the conversation about the right way to do that soon.